imap.lisp

```diff
@@ -293,28 +293,36 @@
     ;; keyword identifying the error (or :unknown)
     :reader po-condition-identifier
     :initform :unknown
-    :initarg :identifier
-    )
+    :initarg :identifier)
    (server-string
     ;; message from the imap server
     :reader po-condition-server-string
     :initform ""
-    :initarg :server-string
-    ))
+    :initarg :server-string)
+   (message
+    :reader po-condition-message
+    :initform ""
+    :initarg :message)
+   (arguments
+    :reader po-condition-arguments
+    :initform nil
+    :initarg :arguments))
```

Comment:
Please don't add `message` and `arguments`: you have changed some places to use these new slots, but kept other places using formerly unavailable `format-control` and `format-arguments`. I have fixed this in https://github.com/orivej/post-office/commit/40a67b8759a8806fd98a508897ebcc7cf5c65d86. Could you rebase?